recipes/uchardet/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -ex
+
+mkdir -p build
+cd build
+cmake -DLIB_INSTALL_DIR="${PREFIX}/lib" \
+  -DCMAKE_BUILD_PREFIX="${PREFIX}" \
+  -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
+  -DCMAKE_BUILD_TYPE=Release \
+  ..
+make -j "${CPU_COUNT}"
+make install
```

Comment:
Is there a `make check`?

Reply:
There isn't.  I tried it and it failed, see https://github.com/conda-forge/staged-recipes/pull/8014/commits/6011f5399d8d66c4662ace13dd9d8e75f55e00fe